vscode/src/chat/chat-view/ChatManager.ts

```diff
@@ -85,13 +85,19 @@ export class ChatManager implements vscode.Disposable {
         await chatProvider?.setWebviewView(view)
     }
 
-    public async executeCommand(command: CodyPrompt, source: ChatEventSource): Promise<void> {
+    public async executeCommand(command: CodyPrompt, source: ChatEventSource, enabled?: boolean): Promise<void> {
         logDebug('ChatManager:executeCommand:called', command.slashCommand)
         if (!vscode.window.visibleTextEditors.length) {
             void vscode.window.showErrorMessage('Please open a file before running a command.')
             return
         }
 
+        if (!enabled) {
+            void vscode.window.showErrorMessage(
+                'This feature has been disabled by your Enterprise instance site administrator.'
```

Comment:
```suggestion
                'This feature has been disabled by your Sourcegraph site administrator.'
```

Suggesting wording improvement, feel free to ignore if someone else said to use the current wording.

Reply:
(We also use "Enterprise Instance" in the UI for login etc)

Reply:
Good thinking folks!!

Reply:
Thanks for the brain dump @slimsag!

@arafatkatze if we want to leave off Enterprise, "your Sourcegraph instance site admin" sounds like the best bet… let's use that.